examples/kserve_deployment/steps/pytorch_steps/pytorch_inference_processor.py

```diff
@@ -27,7 +27,7 @@ class PyTorchInferenceProcessorStepConfig(BaseStepConfig):
 
     img_url: Optional[
         str
-    ] = "https://raw.githubusercontent.com/kserve/kserve/master/docs/samples/v1beta1/torchserve/v1/imgconv/0.png"
+    ] = "https://raw.githubusercontent.com/kserve/kserve/master/docs/samples/v1beta1/torchserve/v1/imgconv/1.png"
```

Comment:
there were two links one for the image of "0" and the other one for "1", I was running an example and showed that we would give the model a picture of "0" while actually the one that had priority was the one for "1" so it was like the model is wrong! to avoid confusion make just pic of one of them